definitions.yaml

```diff
@@ -504,6 +504,9 @@ definitions:
       owner:
         type: string
         description: Name of the organization owning the cluster
+      release:
+        type: string
+        description: The semantic version number
```

Comment:
can we add "of this cluster" to the end of the sentence?

Reply:
Done.